### ISSUE TITLE
Unexpected error while calling npm-groovy-lint

### ISSUE THREAD

**Issue by lewisohn** [nvuillam/vscode-groovy-lint#261]
Not sure if this is an error or if I just don't know how to configure the extension properly.

I'm running VS Code 1.92.2 on Windows 11 version 10.0.26100 with version 3.3.1 of the Groovy Lint, Format and Fix extension (nicolasvuillamy.vscode-groovy-lint). I've added C:\Program Files\nodejs to Windows' `Path` environment variable. `node -v` returns `v22.14.0` and `npm -v` returns `10.9.2`.

`npm-groovy-lint -v` returns the following:

```
GroovyLint: Successfully processed CodeNarc: 
CodeNarc version 3.3.0
npm-groovy-lint version 15.1.0

Embeds:
CodeNarc version 3.3.0
- Groovy version 3.0.9 (superlite)
```

`java -version` returns the following:

```
java version "24" 2025-03-18
Java(TM) SE Runtime Environment (build 24+36-3646)
Java HotSpot(TM) 64-Bit Server VM (build 24+36-3646, mixed mode, sharing)
```

I've restarted VS Code multiple times.

I have a .groovy file open. Every so often, I get an error in VS Code: "There has been an unexpected error while calling npm-groovy-lint. Please join the end of the logs in Output/GroovyLint if you report the issue"

![Image](https://github.com/user-attachments/assets/29899296-8012-44db-af50-1414f0935f4f)

To clarify, this error pops up periodically, even when I'm not actively modifying my open .groovy file. When I invoke the Format Document command in VS Code, it takes a very long time for anything to happen at all before erroring. This log is from a manually invoked format command: [GroovyLint.log](https://github.com/user-attachments/files/19569320/GroovyLint.log)

I'm using the [recommended](https://github.com/nvuillam/npm-groovy-lint/blob/main/lib/.groovylintrc-recommended.json) .groovylintrc-recommended.json configuration file.

I've tried installing npm-groovy-lint both with and without the global flag `-g`, but it made no difference.

Thanks if you can help!

**Comment by verovit** [comment]
I have the same issus, I've been having it since I removed my old java jdk installations and updated to java 21.

**Comment by Axel2293** [comment]
Same issue here.
